#this message file contains all the variables used for the motor driver. this is for only one motor

int64 enable_a  #enable pin for motor A

# direction pins for motor A

int64 motor_a
int64 motor_b

